skewer.yaml

```diff
@@ -0,0 +1,217 @@
+steps:
+  - standard: platform/install_the_skupper_command_line_tool
+  - standard: platform/access_your_kubernetes_clusters
+  - standard: platform/install_skupper_on_your_kubernetes_clusters
+  - title: Create your sites
+    preamble: |
+      A Skupper _site_ is a location where components of your
+      application are running.  Sites are linked together to form a
+      network for your application.  In Kubernetes, a site is associated
+      with a namespace.
+
+      Use the kubectl apply command to declaratively create sites in the kubernetes
+      namespaces. This deploys the Skupper router. Then use kubectl get site to see
+      the outcome.
+
+      **Note:** If you are using Minikube, you need to [start minikube
+      tunnel][minikube-tunnel] before you run `skupper init`.
+
+      [minikube-tunnel]: https://skupper.io/start/minikube.html#running-minikube-tunnel
+    commands:
+      public1:
+        - run: kubectl apply -f ./public1-crs/site.yaml
+        - run: kubectl wait --for condition=Ready --timeout=60s site/public1
+          output: |
+            site.skupper.io/public1 created
+            site.skupper.io/public1 condition met
+      public2:
+        - run: kubectl apply -f ./public2-crs/site.yaml
+        - run: kubectl wait --for condition=Ready --timeout=60s site/public2
+          output: |
+            site.skupper.io/public2 created
+            site.skupper.io/public2 condition met
+      private1:
+        - run: kubectl apply -f ./private1-crs/site.yaml
+        - run: kubectl wait --for condition=Ready --timeout=60s site/private1
+          output: |
+            site.skupper.io/private1 created
+            site.skupper.io/private1 condition met
+  - title: Link your sites
+    preamble: |
+      A Skupper _link_ is a channel for communication between two sites.
+      Links serve as a transport for application connections and
+      requests.
+
+      Creating a link requires use of two `skupper` commands in
+      conjunction, `skupper token issue` and `skupper token redeem`.
+
+      The `skupper token issue` command generates a secret token that
+      signifies permission to create a link.  The token also carries the
+      link details.  Then, in a remote site, The `skupper token
+      redeem` command uses the token to create a link to the site
+      that generated it.
+
+      **Note:** The link token is truly a *secret*.  Anyone who has the
+      token can link to your site.  Make sure that only those you trust
+      have access to it.
+
+      First, use `skupper token issue` in public1 to generate the
+      token.  Then, use `skupper token redeem` in public2 to link the
+      sites.  Using the flag redemptions-allowed specifies how many tokens
+      are created.  In this scenario public2 and private1 will connect to
+      public1 so we will need two tokens.
+    commands:
+      public1:
+        - run: skupper token issue ./public1.token --redemptions-allowed 2
+      public2:
```

Comment:
Something is getting lost in translation as the README.md is missing a few redeems

Reply:
fixed